sigs.yaml

```diff
@@ -1967,19 +1967,19 @@ workinggroups:
       - name: Torin Sandall
         github: tsandall
         company: Styra
-      - name: David Oppenheimer
-        github: davidopp
+      - name: Yisui Hu
+        github: easeway
         company: Google
-      - name: Clayton Coleman
-        github: smarterclayton
+      - name: Erica Von Beulow
```

Comment:
`Erica von Buelow` (lowercase 'v', and 'u' before 'e') 🙂

Reply:
nixed it :)